stl/inc/xutility

```diff
@@ -3444,20 +3514,29 @@ public:
     static constexpr bool _Unwrap_when_unverified = _Do_unwrap_when_unverified_v<iterator_type>;
 
     template <class _Src, enable_if_t<_Wrapped_seekable_v<iterator_type, const _Src&>, int> = 0>
-    constexpr void _Seek_to(const move_iterator<_Src>& _It) {
+    constexpr void _Seek_to(const move_iterator<_Src>& _It) noexcept(noexcept(_Current._Seek_to(_It._Get_current()))) {
         _Current._Seek_to(_It.base());

```

Comment:
`_It.base()` here doesn't agree with `_It._Get_current()` in the _noexcept-specifier_. (I assume you meant to change this line like you did 3523.)